### ISSUE TITLE
Dependency Tree not working since version 1.65

### ISSUE THREAD

**Issue by jkrishnaraj** [javamelody/javamelody#717]
Javamelody configured on a centralized server the dependency tree is not working anymore since version 1.65. Looks a new Class file was introduced HtmlDependenciesReport.class which seems to be causing the issues. This no longer works after version 1.65.0

![image](https://user-images.githubusercontent.com/35076325/34534148-143767f6-f08b-11e7-9515-1669981ab020.png)

  Is there a way to make this work or is this a bug ?

**Comment by evernat** [comment]
It works for me with the latest version.
But you need to upgrade to the latest version, or at least to the same version, in the collector server AND in the monitored webapp.
So, you will see a "Dependencies" link to a new page like you can see in the online demo:
http://javamelody.org/demo/monitoring
http://javamelody.org/demo/monitoring?part=dependencies

**Comment by jkrishnaraj** [comment]
For this to work do I need pom.xml in the war file ?

**Comment by evernat** [comment]
No you don't need pom.xml in the war file for this to work.
What is sure is that your screenshot above is not from a recent version of the javamelody collector server. So you certainly need to upgrade the version of your collector server.

**Comment by jkrishnaraj** [comment]
Yes that is the screen shot from old version, We are currently at 1.69 and it is not working there. I attached that screenshot to show that how it used to work.

**Comment by jkrishnaraj** [comment]
I created a dummy war file with a single jsp file and just few jar files in WEB-INF/lib and the dependencies link is working and it is displaying the jar files,  But it is not working in the actual application. 

So here is what I did to find the root cause. I copied all the jar files from the actual application web-inf/lib to the dummy application web-inf/lib folder and removed any duplicate jar files and it stopped working again in the dummy application. I had a total of 198 jar files in web-inf/lib directory. I randomly started deleting the jar files in the dummy application and then it started working at count 138. So looks it might work somewhere between count 138 and 168. 

Question is, is there a limit on number of jar files it can display?
